content/applications/hr/payroll/configuration.rst

```diff
@@ -0,0 +1,315 @@
+
+.. image:: configuration/payroll-settings.png
+   :align: center
+   :alt: Settings available for payroll.
+
+Any country-specific localizations are set up in the localization section of the settings screen.
+All localization items are pre-populated when the country is specified during the creation of the
+database. It is not recommended to alter the specific localization settings unless specifically
+required.
+
+Work entries
+============
+
+Work entry types
+----------------
+
+Work entries are specified on timesheets when an employee is logging their time, such as `Sick Time
+Off`, `Training`, or `Public Holiday`. Work entries can be configured to account for all types of
+work and time off.
+
+When creating a work entry in the *Payroll* application, or when an employee enters information in
+the *Timesheets* application, a work entry type needs to be selected. This list of work entry types
+is automatically created based off the localization settings set in the database.
+
+Work entry types all have a code to aid in the creation of payslips, and ensure all taxes and fees
+are correctly entered.
+
+Go to :menuselection:`Payroll --> Configuration --> Work Entry Types` to view the current work entry
+types available.
+
+.. image:: configuration/work-entry-types.png
+   :align: center
+   :alt: List of all work entry types currently available.
+
+New work entry type
+~~~~~~~~~~~~~~~~~~~
+
+To create a new :guilabel:`Work Entry Type`, click the :guilabel:`Create` button. Enter the
+information on the form:
+
+- :guilabel:`Work Entry Type Name`: The name should be short descriptive, such as `Sick Time` or
+  `Public Holiday`.
+
+- :guilabel:`Code`: This code appears with the work entry type on timesheets and payslips. Since
+  the code is used in conjunction with the *Accounting* application, it is advised to check with
+  the accounting department for a code to use.
+
+- :guilabel:`Sequence`: The sequence determines the order the work entry is be computed in the
+  payslip list.
+
+- Check Off Boxes: If any of the items in the list applies to the work entry, check off
+  the box by clicking it. If :guilabel:`Time Off` is checked off, a :guilabel:`Time Off Type` field
+  appears. This field has a drop-down menu to select the specific type of time off, or a new type of time
+  off can be entered.
+
+.. image:: configuration/new-work-entry.png
+   :align: center
+   :alt: New work entry type box.
+
+- **Rounding:**
+
+  - **No Rounding:** A time sheet entry is not modified.
+  - **Half Day:** A time sheet entry is rounded to the closest half day amount.
+  - **Day:**  A time sheet entry is rounded to the closest full day amount.
+
+.. example::
+   If an employee enters a time of 5.5 hours on a timesheet, and :guilabel:`Rounding` is set to
+   :guilabel:`No Rounding`, the entry remains 5.5 hours. If :guilabel:`Rounding` is set to
+   :guilabel:`Half Day`, the entry is changed to 4 hours. If it is set to :guilabel:`Day`, it is
```

Comment:
Correct- when set to rounding, it rounds to whatever it was sent, but I have no idea of the legality of it- I just know that's how Odoo works =D 

As for the hours, you are correct- it is based off of what the working day is, which is set on the working hours. So it rounds based off of what a "day" is set to, so I like the addition of the 8 hour day specification.  I just put it at the beginning since both half and whole day is based off the length a "day" is